hivemq-edge/src/frontend/src/extensions/datahub/components/helpers/VersionManagerSelect.tsx

```diff
@@ -0,0 +1,41 @@
+import { labelValue, WidgetProps } from '@rjsf/utils'
+import { getChakra } from '@rjsf/chakra-ui/lib/utils'
+import { FormControl, FormLabel } from '@chakra-ui/react'
+import { CreatableSelect } from 'chakra-react-select'
+import { useTranslation } from 'react-i18next'
+
+export const VersionManagerSelect = (props: WidgetProps) => {
+  const { t } = useTranslation('datahub')
+  const chakraProps = getChakra({ uiSchema: props.uiSchema })
+
+  const options = [
+    { label: 'v0.0', value: 'v0.0' },
```

Comment:
question: Is this a fallback if no data is present?

Reply:
No, its the default data schema in place. Fixing typescript